service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/Message.java

```diff
@@ -110,52 +111,6 @@ public void setUserId(String userId)
         this.userId = userId;
     }
 
-    /**
-    * [Optional] Used when batching on HTTP Default: false.
-    **/
-    private Boolean httpBatchSerializeAsString;
-
-    /**
-    * [Optional] Used when batching on HTTP Default: UTF-8.
-    **/
-    private StandardCharsets httpBatchEncoding;
-
-    /**
-    * [Stamped on servicebound messages by IoT Hub] The authenticated deviceId used to send this message.
-    **/
-    private String connectionDeviceId;
-
-    /**
-    * [Stamped on servicebound messages by IoT Hub] The generationId of the authenticated device used to send this message.
-    **/
-    private String connectionDeviceGenerationId;
-
-    /**
-    * [Stamped on servicebound messages by IoT Hub] The authentication type used to send this message, format as in IoT Hub Specs
-    **/
-    private String connectionAuthenticationMethod;
-
-    /**
-    * [Required in feedback messages] Used in feedback messages generated by IoT Hub.
-    * 0 = success 1 = message expired 2 = max delivery count exceeded 3 = message rejected
-    **/
-    private FeedbackStatusCode feedbackStatusCode;
-
-    /**
-    * [Required in feedback messages] Used in feedback messages generated by IoT Hub. "success", "Message expired", "Max delivery count exceeded", "Message rejected"
-    **/
-    private String feedbackDescription;
-
-    /**
-    * [Required in feedback messages] Used in feedback messages generated by IoT Hub.
-    **/
-    private String feedbackDeviceId;
-
-    /**
-    * [Required in feedback messages] Used in feedback messages generated by IoT Hub.
-    **/
-    private String feedbackDeviceGenerationId;
-
```

Comment:
All of these are not popualted anywhere. Things compile, but if anyone knows why we should keep them, let me know.